gobblin-modules/gobblin-http/src/main/java/gobblin/r2/R2RestResponseHandler.java

```diff
@@ -26,28 +29,37 @@
 public class R2RestResponseHandler implements ResponseHandler<RestRequest, RestResponse> {
 
   public static final String CONTENT_TYPE_HEADER = "Content-Type";
+  private final String R2_RESPONSE_EVENT_NAMESPACE = "gobblin.http.r2.response";
+  private final String R2_FAILED_REQUEST = "R2FailedRequest";
+  private final String REQUEST = "request";
```

Comment:
Move `REQUEST` and `STATUS_CODE` to `HttpConstants`. They may be used by other kinds of response handler, say `HttpResponseHandler`